src/components/inputs/CPriceInput.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <v-text-field
+    v-model="fieldValue"
```

Comment:
Do you know why the jumping occurs if you delete the digit next to the comma? Can we fix that?

Reply:
thats a good question, but I have no idea atm, why this happens. Seems like this should also happen, when deleting something in front of the comma, but it doesn't

Reply:
Could you please check what exactly happens there? Maybe it is not too hard to fix after all

Reply:
I had a look and I think this has something to do with the "hard" change of the format at the point of the cursor. This could possibly be fixed by manually setting the pointers position, but thats also something to do later

Reply:
Okay, then please create an issue.

Reply:
#76

Reply:
#76